src/containers/SearchPage/SearchPage.shared.js

```diff
@@ -458,3 +466,23 @@ export const createSearchResultSchema = (
     },
   };
 };
+
+export const getDatesAndSeatsMaybe = (dates, seats) => {
+  const today = stringifyDateToISO8601(new Date());
+  const aWeekFromNow = stringifyDateToISO8601(addTime(today, 7, 'day'));
+  // Get parameters for dates and seats:
+  const datesAndSeatsMaybe =
+    // If both dates and seats are included, pass both
+    dates && seats
+      ? { dates, seats }
+      : // Dates can be queried without seats
+      dates
+      ? { dates }
+      : // Seats cannot be queried without dates – pass a default date range
```

Comment:
Add the comments above the variable. These inside-the-ternary comments make the optional chaining pretty difficult to read.

Reply:
Comments moved to a clearer formatting!